src/main/java/org/broadinstitute/hellbender/tools/recalibration/covariates/Covariate.java

```diff
@@ -59,5 +62,24 @@
      * @return the maximum value possible for any key representing this covariate
      */
     public int maximumKeyValue();
+
+    /**
+     * Extract the names of the covariates. Strip the "Covariate" because that's how it's stored in the report.
+     */
+    public static List<String> classNameList(StandardCovariateList covs) {
```

Comment:
Why is this a static method here instead of an instance method in `StandardCovariateList` itself?